Examples/JamitFoundationExample/App/Sources/ListView/SampleListViewController.swift

```diff
@@ -0,0 +1,82 @@
+            """
+        )
+        let scrollViewController = ScrollViewController(contentViewController: viewController)
+        navigationController?.pushViewController(scrollViewController, animated: true)
+    }
+}
+
+extension SampleListViewController: UITableViewDelegate {
+    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        switch indexPath.row {
+        case 0:
```

Comment:
Maybe we can use a wrapping ActionView for the items to avoid of using the switch case over the indexPath

Reply:
I would prefer leaving it as is, as otherwise it might be too much at a time which might be confusing as well. When showing an example we should focus on the necessary things and the `ActionView` is already implemented in another scenario. 

Thanks for noticing as it would be a more compact way of writing and we would not need to be delegated by the `UITableView` which furthermore removes some complexity.